### ISSUE TITLE
Bottom of report content covered by footer

### ISSUE THREAD

**Issue by dan-kelleher-os** [datapane/datapane#155]
In Datapane version 0.12.0 when a report spans the entire browser window height, the content at the bottom of the report will be covered by the footer. This seems to apply to all Datapane elements.

Example

```python
import datapane as dp
import pandas as pd

parts = []
for i in range(15):
    table = dp.Table(pd.io.formats.style.Styler(pd.DataFrame([{"Field 1": "field 1 text", "Field 2": "field 2 text", "Field 3": "field 4 text", "row number": i,}])).hide_index())
    parts.append(table)
dp.Report(*parts).save("dp_example.html")
```
will result in this at the bottom of the report (row 14 is not displayed and should be)
![image](https://user-images.githubusercontent.com/85510380/133837765-754ebfb0-f468-4780-ab48-ca60daa1d3b1.png)

**Comment by mands** [comment]
@dan-kelleher-os Thanks for raising and the reproducible example - this should be a simple fix and we'll try to get it out shortly.

**Comment by HaasCP** [comment]
Are there any news to that issue? We have similar problems since we create reports with >50 pages. With so many pages, we cannot navigate between pages using the header with the page tabs. Either we would need a scroll bar for the page tabs or the footer is required to be uncovered by the datapane panel. Best solution would probably be to solve both.

Thanks a lot in advance! Looking forward to hearing any updates!

**Comment by HaasCP** [comment]
Just discovered, that this issue is solved in datapane verion 0.13.2. It is a beauty, thanks very much!
